### PR TITLE
Add BestRes Latest

### PR DESCRIPTION
**Pull request by decors**

### CODE REVIEW

File: Casks/bestres.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'bestres' do
+  version 'latest'
+  sha256 '1b706a3550edbc1411bdc4665490cce3b22cef68a6abae82af0fbd46370ce697'
+
+  url 'http://dl.devmate.com/com.icyberon.BestRes/BestRes.zip'
+  name 'BestRes'
+  homepage 'http://icyberon.com/bestres/'
+  license :unknown
```
Comment (vitorgalvao) [inline Casks/bestres.rb]: `:gratis`.

File: Casks/bestres.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'bestres' do
+  version 'latest'
```
Comment (vitorgalvao) [inline Casks/bestres.rb]: We haven’t used `'latest'` in this way in a long time.

File: Casks/bestres.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  sha256 '1b706a3550edbc1411bdc4665490cce3b22cef68a6abae82af0fbd46370ce697'
```
Comment (vitorgalvao) [inline Casks/bestres.rb]: Setting a shasum with a `:latest` cask makes no sense.

File: Casks/bestres.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+  url 'http://dl.devmate.com/com.icyberon.BestRes/BestRes.zip'
```
Comment (vitorgalvao) [inline Casks/bestres.rb]: Needs a `# devmate.com is the official download host per the vendor homepage` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5)).

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in f0dbcff9d4ea0066b9a8414d65402c5a0a981be8. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by decors** [comment]
@vitorgalvao Thanks for your feedback and fix.

**Pull request closed without merging**